go/vt/discovery/topology_watcher.go

```diff
@@ -217,6 +214,10 @@ func (tw *TopologyWatcher) loadTablets() {
 	tw.mu.Lock()
 
 	for alias, newVal := range newTablets {
+		if !(tw.tabletFilter == nil || tw.tabletFilter.IsIncluded(newVal.tablet)) {
```

Comment:
Nit, and I know this is what we were previously using, but IMO this is easier to read and process as a human:
```
if tw.tabletFilter != nil && !tw.tabletFilter.IsIncluded(newVal.tablet)) {
```

Reply:
haha -- i had to apply demorgan's law to figure out what this was doing as well. will update